tests/expr/fbinary/test-truediv.py

```diff
@@ -116,21 +126,21 @@ def test_div_floats_random(seed):
 def test_mul_booleans_integers_floats_random(seed):
```

Comment:
Hmm, shouldn't the test name be test_**div**_booleans_integers_floats_random?

Reply:
Yeah, probably I had a typo in the test name when created. @pradkrish if in this PR you could rename it to `test_div_booleans_integers_floats_random` I would greatly appreciate it. Thanks.

Reply:
Done